Update faq.yml

bengimblett opened the pull request:
Adding customer related question to help others clarify that Service Endpoint are not impacted by the addition of a VNet NAT GW

bengimblett committed:
Update faq.yml

Adding customer related question to help others clarify that Service Endpoint are not impacted by the addition of a VNet NAT GW

aimee-littleton commented on articles/virtual-network/nat-gateway/faq.yml:
```diff
@@ -90,7 +90,11 @@ sections:
+
+      - question: Can I use Virtual Network NAT gateway with Service Endpoints?
+        answer: | 
+           Yes. The addition of a Virtual Network NAT Gateway to a subnet with Service Endpoints does not affect the Service Endpoints. Service Endpoints enable a more specific route for the destination Azure service traffic they represent. Traffic for the Service Endpoint will continue to be routed toward the service and wont go via the NAT Gateway.
```
@bengimblett could we also maybe add a link to the service endpoint doc so that customers can read more there if they  want? 

Something like:
To learn more about service endpoints, see Service Endpoint Overview. 

And link to this doc here: https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-service-endpoints-overview

bengimblett replied:
updated :)

bengimblett committed:
Update faq.yml

Updated to add service endpoint link - and changed the case for "service endpoints" for consistency to match the lower case in the service endpoints doc

ktoliver committed:
PR review edit, relative link
